resources/templates/ansic_cmake_connectionstring/Readme.md

```diff
@@ -1,24 +1,24 @@
 # Use generated code in your device project
 
-One of the features VS Code Digital Twin tooling provides is generating stub code based on Device Capability Model (DCM) you specified.
+One of the features VS Code Digital Twin tooling provides is generating stub code based on the Device Capability Model (DCM) you specified.
 
 Follow the steps to use the generated code with the Azure IoT Device C SDK source to compile a device app.
 
-For more details about setting up development environment for compiling the C Device SDK. Check the [instructions](https://github.com/Azure/azure-iot-sdk-c/blob/master/iothub_client/readme.md#compiling-the-c-device-sdk) for each platform.
+For more details about setting up your development environment for compiling the C Device SDK. Check the [instructions](https://github.com/Azure/azure-iot-sdk-c/blob/master/iothub_client/readme.md#compiling-the-c-device-sdk) for each platform.
 
 ## Windows
 
-1. Install [Visual Studio 2017](https://www.visualstudio.com/downloads/). You can use the Visual Studio Community Free download if you meet the licensing requirements. (Visual Studio 2015 is also supported.)
+1. Install [Visual Studio](https://www.visualstudio.com/downloads/). You can use the Visual Studio Community Free download if you meet the licensing requirements. (Visual Studio 2015 is also supported.)
 
-    > Be sure to include Visual C++ and NuGet Package Manager.
+    > Be sure to include **Desktop development with C++** and **NuGet Package Manager** from the Visual Studio Installer.
```

Comment:
and **NuGet Package Manager** workloads from...

Reply:
NuGet Package Manager doesn't belongs to workloads, it is one of the Individual component